client/src/pages/SignAll.js

```diff
@@ -91,13 +136,13 @@ export default function SignAll() {
   }
 
   const handleChange = type => event => {
-    console.log("event is ", type, event);
     if (type === "email") {
       setEmailError(false);
       setEmail(event.target.value);
     }
     if (type === "password") {
       setPassowrdError(false);
+      //Do we need password validation like empty spaces,  strength of passoword and not allow special characters?
```

Comment:
I think we should just do a limit on the length (like at least 6 characters). I'm not a fan of too strong password validation (for these projects, and in general, but that's a separate discussion)

Reply:
got it.